src/mlpack/methods/ann/layer/reparametrization.hpp

```diff
@@ -39,7 +39,8 @@ namespace ann /** Artificial Neural Network. */ {
  *              Xavier Glorot, Matthew Botvinick, Shakir Mohamed and
  *              Alexander Lerchner | Google DeepMind},
  *   journal = {2017 International Conference on Learning Representations(ICLR)},
- *   year    = {2017}
+ *   year    = {2017},
+ *   url     = {https://openreview.net/references/pdf?id=Sy2fzU9gl}
```

Comment:
Is this a stable/permanent url?

Reply:
The google deepmind website ( https://deepmind.com/research/publications/beta-VAE-Learning-Basic-Visual-Concepts-with-a-Constrained-Variational-Framework ) leads to this url, so I believe it should be stable .

Reply:
Maybe we should use `https://deepmind.com/research/publications/beta-VAE-Learning-Basic-Visual-Concepts-with-a-Constrained-Variational-Framework` instead?

Reply:
Okay @zoq .